share/completions/which.fish

```diff
@@ -4,9 +4,9 @@ if which -v >/dev/null 2>/dev/null # GNU
     complete -c which -l skip-alias -d "Ignore option '--read-alias'"
     complete -c which -l read-functions -d "Read shell function definitions from stdin, reporting matching ones on stdout"
     complete -c which -l skip-functions -d "Ignore option '--read-functions'"
-    complete -c which -l skip-dot -d "Skip directories in PATH that start with a dot"
-    complete -c which -l skip-tilde -d "Skip directories in PATH that start with a tilde and executables which reside in the HOME directory"
-    complete -c which -l show-dot -d "If a directory in PATH starts with a dot and a matching executable was found for that path, then print './programname'"
+    complete -c which -l skip-dot -d "Skip directories in PATH starting with a dot"
+    complete -c which -l skip-tilde -d "Skip directories in PATH starting with a tilde and executables in the HOME directory"
+    complete -c which -l show-dot -d "If a directory in PATH starts with a dot and a matching executable was found, print './programname'"
```

Comment:
"For matches in PATH dirs that start with a dot, print './programname'" is shorter